Automattic/wp-calypso
# Site Settings: Remove Disabled Input for WordPress Version

#### Changes proposed in this Pull Request

* Removes the disabled input in Site Settings for the WordPress version

> Some of the elements to mention should obviously not be a text input (why would the version number be a text input?)

#### Testing instructions

On a Jetpack connected site, visit `/settings/general` and check the WordPress version.

**Current:**

<img width="766" alt="Screenshot 2019-04-21 at 14 03 43" src="https://user-images.githubusercontent.com/43215253/56470706-281aaf00-6441-11e9-84e4-54c705e72321.png">

**Proposed:**

<img width="771" alt="Screenshot 2019-04-21 at 14 03 21" src="https://user-images.githubusercontent.com/43215253/56470708-336dda80-6441-11e9-9795-8b7e9c66120c.png">

cc @simison, @drw158 

See #31207 and #26982 for some context on this
